src/mam4xx/seq_drydep.hpp

```diff
@@ -137,6 +137,168 @@ void set_hcoeff_scalar(const Real sfc_temp, Real heff[]) {
   }
 }
 
+inline Data set_gas_drydep_data() {
+  using View1D = typename DeviceType::view_1d<Real>;
+  using View2D = typename DeviceType::view_2d<Real>;
+  using View1DHost = typename HostType::view_1d<Real>;
+
+  using ViewBool1D = typename DeviceType::view_1d<bool>;
+  using ViewInt1D = typename DeviceType::view_1d<int>;
+  using ViewBool1DHost = typename HostType::view_1d<bool>;
+  using ViewInt1DHost = typename HostType::view_1d<int>;
+
+  // allocate the views
+  constexpr int n_drydep = 3;
+  constexpr int gas_pcnst = mam4::gas_chemistry::gas_pcnst;
+  Data data{
+      View1D("drat", n_drydep),
+      View1D("foxd", n_drydep),
+      View2D("rac", NSeas, NLUse),
+      View2D("rclo", NSeas, NLUse),
+      View2D("rcls", NSeas, NLUse),
+      View2D("rgso", NSeas, NLUse),
+      View2D("rgss", NSeas, NLUse),
+      View2D("ri", NSeas, NLUse),
+      View2D("rlu", NSeas, NLUse),
+      View2D("z0", NSeas, NLUse),
+      ViewBool1D("has_dvel", gas_pcnst),
+      ViewInt1D("map_dvel", gas_pcnst),
+      -1, // FIXME: so2_ndx
+  };
+
+  // populate the views with data taken from mam4 validation test data
+  Real drat_a[n_drydep] = {1.3740328303634228, 2.3332297194282963,
+                           1.8857345177418792};
+  View1DHost drat_h(drat_a, n_drydep);
+  Kokkos::deep_copy(data.drat, drat_h);
+
+  Real foxd_a[n_drydep] = {1.0, 1e-36, 1e-36};
+  View1DHost foxd_h(foxd_a, n_drydep);
+  Kokkos::deep_copy(data.foxd, foxd_h);
+
+  // clang-format off
+  Real rac_a[NSeas][NLUse] = { 
+     { 100.0, 200.0, 100.0,2000.0,2000.0,2000.0,   0.0,   0.0, 300.0, 150.0, 200.0}, 
+     { 100.0, 150.0, 100.0,1500.0,2000.0,1700.0,   0.0,   0.0, 200.0, 120.0, 140.0},
+     { 100.0,  10.0, 100.0,1000.0,2000.0,1500.0,   0.0,   0.0, 100.0,  50.0, 120.0}, 
+     { 100.0,  10.0,  10.0,1000.0,2000.0,1500.0,   0.0,   0.0,  50.0,  10.0,  50.0}, 
+     { 100.0,  50.0,  80.0,1200.0,2000.0,1500.0,   0.0,   0.0, 200.0,  60.0, 120.0} };
+  for (int i = 0; i < NSeas; ++i) {
+    for (int j = 0; j < NLUse; ++j) {
+        data.rac(i, j) = rac_a[i][j];
+    }
+  }  
```

Comment:
The way of initialization works on CPU but probably not for GPU. Need to fix it. @odiazib @singhbalwinder and I discussed it before, I @odiazib has ways to fix it.